uuid/v1.ts

```diff
@@ -136,6 +136,10 @@ export function generate(
     throw new Error("Can't create more than 10M uuids/sec");
   }
 
+  if (node.length < 6) {
+    throw new Error("Uuid node needs to have at least 6 numbers");
+  }
```

Comment:
`node` can be passed in as a parameter, but if empty it will be set to an array of 6 numbers. All the current tests call this function with ether no `node` or a `node` of 6 numbers. I think its safe to say that calling this function with a `node` of less than 6 numbers is undefined behaviour and we should be safe to throw.

Reply:
https://github.com/denoland/deno_std/blob/f0401ad2a6535d7b47147ffd9322edd603bc780b/uuid/v1.ts#L34-L39